pkg/cephfs/nodeserver.go

```diff
@@ -150,7 +150,7 @@ func (*NodeServer) mount(volOptions *volumeOptions, req *csi.NodeStageVolumeRequ
 
 	klog.V(4).Infof("cephfs: mounting volume %s with %s", volID, m.name())
 
-	if err = m.mount(stagingTargetPath, cr, volOptions, volID); err != nil {
```

Comment:
so this is unused param is found by golangci?

Reply:
yes it  found during static check by golangci